### ISSUE TITLE
Intermittent unit test `test_accessibility_after_navigate_and_back`

### ISSUE THREAD

**Issue by yezhizhen** [servo/servo#44065]
```
thread 'test_accessibility_after_navigate_and_back' (10772) panicked at C:\Users\Administrator\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\accesskit_consumer-0.35.0\src\tree.rs:123:17:
    Cannot push subtree TreeId(879211fb-8799-5492-9a31-95a35c05a192): no graft node exists for this tree. Push the graft node (with tree_id property set) before pushing the subtree.
    stack backtrace:
       0: std::panicking::panic_handler
                 at /rustc/ded5c06cf21d2b93bffd5d884aa6e96934ee4234/library\std\src\panicking.rs:698
       1: core::panicking::panic_fmt
                 at /rustc/ded5c06cf21d2b93bffd5d884aa6e96934ee4234/library\core\src\panicking.rs:80
       2: accesskit_consumer::tree::State::update
       3: accesskit_consumer::tree::Tree::update_and_process_changes
       4: accessibility::main
       5: core::ops::function::FnOnce::call_once
                 at /rustc/ded5c06cf21d2b93bffd5d884aa6e96934ee4234/library\core\src\ops\function.rs:250
    note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

  TRY 4 FAIL [   0.941s] (499/893) servo::accessibility test_accessibility_after_navigate_and_back
```

[Run](https://github.com/servo/servo/actions/runs/24184931983/job/70587335987)

Introduced in #42338
cc @alice @delan

**Comment by Loirooriol** [comment]
Also in linux: https://github.com/servo/servo/actions/runs/24187224165/job/70594927112

**Comment by Loirooriol** [comment]
But I took #42338 out of the merge queue, so I think this can be closed
